source/Track.cpp

```diff
@@ -77,6 +77,8 @@ const std::string Track::GetTitle(GameState state) const
 		case GameState::LANDED:
 			return landedTitle;
 			break;
+		default:
```

Comment:
You need to return something here, or the g++ builds will keep failing.

Reply:
there you go